aten/src/ATen/native/TensorFactories.h

```diff
@@ -106,7 +106,7 @@ inline Tensor& fill_empty_deterministic_(Tensor& tensor) {
     AT_DISPATCH_V2(
       tensor.scalar_type(), "fill_empty_deterministic_", AT_WRAP([&]() {
         tensor.fill_(std::numeric_limits<scalar_t>::quiet_NaN());
-    }), AT_EXPAND(AT_FLOATING_TYPES), AT_EXPAND(AT_COMPLEX_TYPES), AT_EXPAND(AT_FLOAT8_TYPES), kBFloat16, kHalf);
+    }), AT_EXPAND(AT_FLOATING_TYPES), AT_EXPAND(AT_COMPLEX_TYPES), AT_EXPAND(AT_FLOAT8_TYPES), kBFloat16, kHalf, kComplexHalf);
```

Comment:
The complication here is I m not sure if kComplexHalf is supported on CPU...

Reply:
![image](https://github.com/user-attachments/assets/17ab361c-d1e1-4bcb-941b-be7135ea2576)

I tried it in my local, but I didn't check the code details that how complex works on CPU.